content/blog/2021/09/2021-09-21-jenkins-at-devops-world.adoc

```diff
@@ -0,0 +1,78 @@
+---
+:layout: post
+:title: "Join Jenkins at DevOps World 2021"
+tags:
+- events
+- conference
+- cicd
+- talks
+- workshops
+author: alyssat
+description: >
+  DevOps World 2021 will include significant Jenkins workshops, sessions, and interactions.
+  Join us September 28-30, 2021 for Jenkins workshops, presentations, and conversations.
+opengraph:
+  image: /images/conferences/devops-world-2021.png
+links:
+  discourse: true
+---
+
+link:https://www.devopsworld.com/[DevOps World] has been the largest gathering for Jenkins for many yearss.
+In keeping with tradition, many Jenkins presentations and sessions are planned for this year's event.
+
+image:/images/conferences/devops-world-2021.png[role="center", align="center", link="https://www.devopsworld.com/"]
+
+Join us for DevOps World on September 28 - 30, 2021.
+The event is virtual, free to attend and will consist of the following Jenkins activities:
```

Comment:
```suggestion
The event is virtual, free to attend and will include the following Jenkins activities:
```
"consist of" sounds like the list is comprehensive, and you certainly don't want to try to list everything that will happen at DevOps World ;-)